tests/Test.ADAL.NET.Integration/SovereignCloudsTests.cs

```diff
@@ -42,6 +44,8 @@ public void Initialize()
                 // validate that authorizationUri passed to WebUi contains instance_aware query parameter
                 new Dictionary<string, string> {{"instance_aware", "true"}});
 
+            HttpMessageHandlerFactory.AddMockHandler(MockHelpers.CreateInstanceDiscoveryMockHandler());
+            HttpMessageHandlerFactory.AddMockHandler(MockHelpers.CreateInstanceDiscoveryMockHandler());
```

Comment:
Is there a reason to have 2 mock calls for this?

Reply:
It won't pass otherwise. It took me quite a while to figure this out. I believe it is because the sovereign feature would trigger one extra instance discovery for a different sovereign.